config/config.go

```diff
@@ -63,6 +63,8 @@ func setConfigDefaults(v *viper.Viper) {
 	v.SetDefault("request_limits.max_size_bytes", 10*1024)
 	v.SetDefault("request_limits.max_num_values", 10)
 	v.SetDefault("request_limits.max_ttl_seconds", 3600)
+	v.SetDefault("routes.index_response", "This application stores short-term data for use in Prebid.")
```

Comment:
How about moving the `index_response` up a level from `routes.index_response` since it's really no longer configuring the routes.

Reply:
Makes sense. Done.